database/console/model_make_command.go

```diff
@@ -0,0 +1,65 @@
+package console
+
+import (
+	"errors"
+	"os"
+	"strings"
+
+	"github.com/goravel/framework/contracts/console"
+	"github.com/goravel/framework/contracts/console/command"
+	"github.com/goravel/framework/support/file"
+	"github.com/goravel/framework/support/str"
+
+	"github.com/gookit/color"
```

Comment:
Nitpick: The third package should be between the system package and `goravel/framework`.